server.js

```diff
@@ -92,8 +93,9 @@ var staticServer = new nodeStaticAlias.Server(STATIC_DIR,{
 	],
 });
 
-httpServer.listen(8034);
-
+httpServer.listen(PORT, () => {
+	console.log(`The app is running on http://localhost:${PORT}`);
```

Comment:
**SAND 🏖**

This is just a suggestion, you could add colors and underline to this console. That way, it looks nice on the terminal and is clickable. here is the colors package https://www.npmjs.com/package/colors

<img width="1262" alt="Screenshot 2022-07-06 at 08 54 13" src="https://user-images.githubusercontent.com/37575223/177487884-7a84a79e-302c-40d3-9d8e-2a4ada89bf14.png">

Reply:
It's clickable already.

There are many ways to add colors but that seems like an overkill right now.